### PR TITLE
chore: improve auto labeling

### PR DESCRIPTION
**Pull request by mcdurdin**
Cherry-picked from #7287.

@keymanapp-test-bot skip

### CODE REVIEW

File: .github/multi-labeler.yml [hunk 1]
```diff
@@ -20,26 +22,38 @@ labels:
...
+  # additional meta flags
+
+  # note, this does not pick up chained PRs automatically
+  - label: 'stable'
+    matcher:
+      baseBranch: '^stable-.+'
+
+  - label: 'cherry-pick'
+    matcher:
+      title: '(🍒|:cherries:)'
```
Comment (jahorton) [inline .github/multi-labeler.yml:44]: Could maybe add one for...

```
label: 'feature-branch'
matcher:
  branch: 'feature-.*'
```

Or something like that?  Just a suggestion, though.

Comment (mcdurdin) [reply .github/multi-labeler.yml:49]: Good idea, missed that one. Done!

**Review by darcywong00 — APPROVED** [review]
lgtm

**Review by jahorton — APPROVED** [review]

**Review by ermshiperete — APPROVED** [review]

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 16.0.66-alpha](https://keyman.com/downloads/releases/alpha/16.0.66)

**Pull request merged**